src/Framework/AspNetCoreAnalyzers/test/Verifiers/CSharpAnalyzerVerifier.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+
+namespace Microsoft.AspNetCore.Analyzers.Verifiers;
+
+public static partial class CSharpAnalyzerVerifier<TAnalyzer>
```

Comment:
Non-blocking feedback: while we're making changes here, do you mind deleting the other verifiers in this directory and updating all the tests to use this one? I realized that with our test setup there was no need for us to have a verifier per analyzer type.

Reply:
Done